corehq/apps/export/models/new.py

```diff
@@ -2311,14 +2317,15 @@ def _get_stock_items_from_question(question, app_id, app_version, repeats):
 
     @classmethod
     def _process_apps_for_export(cls, domain, schema, identifier, app_build_ids, task,
-                                 for_new_export_instance=False):
+                                 for_new_export_instance=False, is_identifier_case_type=False):
```

Comment:
I don't love that we are adding a case export specific parameter to every ExportDataSchema subclass, but also recognize that you are working within the limitations of the current class inheritance setup. It does look like the implementations of `_process_apps_for_export` in the FormExportDataSchema and SMSExportDataSchema just call the super class, so we could at least remove those to reduce the boilerplate feeling of this change. Does that sound right to y'all @biyeun and @AmitPhulera?

Reply:
@gherceg how's the latest change looking? I removed the overrides on `FormExportDataSchema` and `SMSExportDataSchema`

Reply:
Thanks for getting back to this Biyeun. That looks better.